### PR TITLE
build: remove ceph-disk-udev entirely 

### PR DESCRIPTION
**Pull request by zealoussnow**
Signed-off-by: Leo Zhang <nguzcf@gmail.com>

### CODE REVIEW

**Comment by zealoussnow** [comment]
ceph-disk-udev is not used now.

**Comment by tchaikov** [comment]
@zealoussnow could you explain the rationale behind this change in the commit message?

**Comment by tchaikov** [comment]
@zealoussnow this rule is supposed to along with ceph-disk. have you tested ceph-disk as well? or, i'd recommend you read through ceph-disk code to understand the parttypeuuid is used by it.

**Comment by zealoussnow** [comment]
@tchaikov I tested ceph-disk right now, it can work without ceph-disk-udev. And I found the parttypeuuid in /dev/disk is generated by rule `60-ceph-by-parttypeuuid.rules`.

**Comment by tchaikov** [comment]
@zealoussnow  thanks for testing it. i don't want to block you, but as you may have already figured out that i am not familiar with this area =(. probably we need to wait @dachary for more insights.

**Comment by zealoussnow** [comment]
@tchaikov OK, thank you.

**Comment by zealoussnow** [comment]
@dachary Thank you, I finished it now.

**Comment by dachary** [comment]
Also please remove ceph-disk-udev itself and squash the two commits into a single one. Thanks !

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -944,7 +944,6 @@ install(PROGRAMS
 install(PROGRAMS
   ${CMAKE_SOURCE_DIR}/src/ceph-create-keys
 #  ${CMAKE_SOURCE_DIR}/src/ceph-disk
-  ${CMAKE_SOURCE_DIR}/src/ceph-disk-udev
```
Comment (tchaikov) [inline src/CMakeLists.txt]: can we just remove `src/ceph-disk-udev` in our source tree, as it is not used anymore?

**Comment by zealoussnow** [comment]
@dachary @tchaikov All is OK.

**Comment by yuriw** [comment]
http://pulpito.ceph.com/yuriw-2017-05-26_19:34:44-rados-wip-yuri-testing_2017_5_27-distro-basic-smithi/

http://pulpito.ceph.com/yuriw-2017-05-29_22:24:45-rados-wip-yuri-testing_2017_5_27---basic-smithi/

**Pull request merged**